html/pfappserver/lib/pfappserver/Form/Config/Provisioning/mobileconfig.pm

```diff
@@ -18,13 +19,20 @@ has_field 'company' =>
    type => 'Text',
   );
 
+has_field 'description' =>
+  (
+   type => 'Text',
+   messages => { required => 'Please specify the Description Provisioning entry.' },
```

Comment:
*description of the provisioning entry

- Description and Provisioning to lowercase
- Missing "of the"